cvmfs/platform_osx.h

```diff
@@ -265,6 +265,8 @@ inline double platform_monotonic_time_ns() {
   return val_ns;
 }
 
+inline uint64_t platform_time() { return platform_monotonic_time_ns(); }
```

Comment:
That would be ticks since boot time, not realtime.  We can simply use C's `gettimeofday()`.

Reply:
`gettimeofday()` is obsolete in Posix 2008. So I was guessing that we should not use it...

What do you consult for OSX documentation?